frontend/src/js/store/organizationSlice/thunks.ts

```diff
@@ -240,6 +243,16 @@ export const editTenantDeviceLimit = createAsyncThunk(`${sliceName}/editDeviceLi
       ]);
     });
 });
+export const editBillingProfile = createAsyncThunk(
+  `${sliceName}/editBillingProfileEmail`,
+  ({ billingProfile }: { billingProfile: BillingProfile }, { dispatch }) => {
+    return Api.patch(`${tenantadmApiUrlv2}/billing/profile`, { ...billingProfile })
```

Comment:
```suggestion
    return Api.patch(`${tenantadmApiUrlv2}/billing/profile`, billingProfile)
```
I don't think this needs to be destructured to be sent